md/database-configuration.md

```diff
@@ -50,9 +44,9 @@ We recommend that you use the WildFly bundle provided by Bonitasoft.
 
 <a id="database_creation" />
 
-## Create the database(s)
+### Create the database(s)
 
-The first step in configuring Bonita BPM with your RDBMS is to create one or two new database(s) (i.e. new schema(s)).
+The first step in configuring Bonita BPM with your RDBMS is to create the new databases (i.e. schemas): one for engine data and platform configuration, another one, optionally, if you use business data. 
```

Comment:
Again, no need to loose the user with extra information, I would remove `and platform configuration`.
We already mentioned it above, that's enough.